github/files/drupal7/scripts/devops/reinstall/sql_workflow.yml

```diff
@@ -6,7 +6,7 @@
 # - Running database updates
 - name: Droping drupal database
   mysql_db: name={{ mysql_db }} state=absent login_user={{ mysql_user }} login_password={{ mysql_pass }}
-  when: pp_environment == "demo" or pp_environment == "default"
+  when: pp_environment == "demo" or pp_environment == "default" or pp_environment == 'bt-dev' or pp_environment == 'bt-demo'
```

Comment:
yeah, this is not good. We can't have different environment names for every CMF.
We need to refactor for been used only basic 4 environments